api/v1beta1/kogitosupportingservice_types.go

```diff
@@ -107,14 +107,14 @@ func (k *KogitoSupportingService) GetSupportingServiceStatus() api.KogitoSupport
 type KogitoSupportingServiceList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []KogitoSupportingService `json:"items"`
+	Items           []*KogitoSupportingService `json:"items"`
```

Comment:
Not sure if changing the default kubebuilder structure is a good idea. Can you try a different approach?

Reply:
done